docs/API Documentation.md

```diff
@@ -408,10 +409,33 @@ PrettyPageHandler::handle()
 
 ## <a name="fn-ajax"></a> `Whoops\isAjaxRequest()`
  #=> boolean
- 
+
 ```php
 // Use a certain handler only in AJAX triggered requests
 if (Whoops\isAjaxRequest()){
   $run->addHandler($myHandler)
 }
+```
+
+## <a name="fn-cli"></a> `Whoops\isCommandLine()`
+ #=> boolean
+
+```php
+// Use a certain handler only in php cli
+if (Whoops\isCommandLine()){
+  $run->addHandler($myHandler)
+}
+```
+
+```php
+/*
+Output the error message only if using command line.
```

Comment:
I added this sample, as it 1:1 reflects the behavior of the deleted `outputOnlyIfCommandLine`